content/tr/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Cloud Native Sözlüğü"
+status: Completed
+---
+
+**Cloud Native Sözlüğü**
+
+Cloud Native Sözlüğü, karmaşıklığı ile ünlü olan cloud native alanının yalnızca teknoloji uzmanları için değil, herkes için anlaşılabilir olmasını hedeflemektedir. Basitliğe odaklanan sözlük, CNCF Business Value Subcommittee (BVS) tarafından yönetilen bir projedir.
+
+**Katkıda Bulunma**
+
+Cloud Native Sözlüğü, ekleme, değişiklik veya iyileştirme yaparak katkıda bulunmak isteyen herkese açıktır. Paylaşılan sözlüğün iyileştirilmesi ve geliştirilmesi için CNCF tarafından yönetilen topluluk odaklı bir süreç benimsenmiştir. Bu sözlük, cloud native teknolojileri çerçevesinde ortak bir kelime dağarcığı oluşturmak amacıyla şirketlerden bağımsız bir platform sağlar. Projenin amacına ve tüzüğüne bağlı kalan tüm katılımcıların katkıları memnuniyetle karşılanmaktadır.
+
+Katkıda bulunmak isteyen herkes, bir GitHub issue veya pull request oluşturabilir. Lütfen [Stil Kılavuzu](https://github.com/cncf/glossary/blob/main/style-guide)'nu inceleyip takip ettiğinizden, [Nasıl Katkıda Bulunabilirim?](https://github.com/cncf/glossary/blob/main/contribute) dokümanını okuduğunuzdan, [CNCF'in Slack topluluğuna](https://communityinviter.com/apps/cloud-native/cncf) katıldığınızdan ve [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) kanalına dahil olduğunuzdan emin olun. Ayrıca Cloud Native Sözlüğü'nü kendi ana diline çevirmek isteyenler için [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) isimli bir kanal da bulunmaktadır. Türkçe çeviriye katkıda bulunmak için [#glossary-localizations-turkish](https://cloud-native.slack.com/archives/C05ATBGLHJ7) kanalına dahil olabilirsiniz.
```

Comment:
We should update these links later, when we have the localizations of these pages.

Don't worry about it for now though.

Reply:
Let me make a note in our index sheet for these. Thanks.